src/less/pricingEstimate.less

```diff
@@ -80,7 +81,8 @@
 
   #priceEstimateContainer {
     box-shadow: 7px 0 7px -7px rgba(0, 0, 0, 0.2);
-
+    position: sticky;
```

Comment:
We should test it out with the monocle mock ups for each app and make sure. Since it will be a minor version update I want to make sure there is no breaking changes